client/src/EstimatesComparison/EstimatesComparison.js

```diff
@@ -248,6 +248,18 @@ class EstimatesExplorer extends React.Component {
       col_click,
     };
 
+    !this.state.show_inactive &&
+      _.forEach(
+        root.children,
+        (org) =>
+          (org.children = _.reject(
```

Comment:
This will cause mutations..! I suggest below

```
    const filtered_root = this.state.show_inactive
      ? root
      : {
          ...root,
          children: 
            _.map(root.children, (org) => ({
              ...org,
              children: _.reject(
                org.children,
                (estimate) =>
                  estimate.data.current_value === 0 &&
                  estimate.data.percent_value === -Infinity
              ),
            })),
        };
```